### PR TITLE
Fix #6373

### LINKED ISSUE #6373: Add support for Moes RGB+CCT Smart Downlight `TS0505B` / `_TZ3210_s9lumfhn`
Please add support for these nice downlights.

Sadly, I do not know how to test any code changes locally.

All I know are the following data points:
- Zigbee model: `TS0505B`
- Manufacturer name: `_TZ3210_s9lumfhn`
- Description: `RGB+CCT Smart Downlight`
- White Label: `Moes`
- Product: `...`

I will make a PR, but it has just `xxxxx` for the product

### PR DESCRIPTION
**Pull request by vanHoesel**
Add support for Moes RGB+CCT Smart Downlight TS0505B / _TZ3210_s9lumfhn

### CODE REVIEW

**Comment by vanHoesel** [comment]
If I knew how to test these changes on my RaspberryPi, I would have done so.
If there is a better Product ID than `xxxxx` I'd happily update this MR

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -912,6 +912,8 @@ const definitions: Definition[] = [
             tuya.whitelabel('Nous', 'P3Z', 'Smart light bulb', ['_TZ3210_cieijuw1']),
             tuya.whitelabel('Moes', 'ZLD-RCW_1', 'RGB+CCT Zigbee LED controller', ['_TZ3000_7hcgjxpc']),
             tuya.whitelabel('Moes', 'ZLD-RCW_2', 'RGB+CCT LED bulb', ['_TZ3210_rcggc0ys']),
+            tuya.whitelabel('Moes', 'xxxxx', 'RGB+CCT Smart Downlight', ['_TZ3210_s9lumfhn']),
```
Comment (Koenkk) [inline src/devices/tuya.ts:915]: Can you use the real model number (listed on the device/product page) instead of `xxxx` here?

Comment (vanHoesel) [reply src/devices/tuya.ts:915]: Hey @Koenkk ,

There was nothing on the box, not om the label at the back of the fixture, nothing on AliExpresss where I bought those...

I have found something on the big interweb: [Moes House](https://moeshouse.com/products/zigbee-smart-light-zb-lzd7-rcw-en?variant=39331508125777), I found a SKU for the 10W, but also for different power consumptions
- ZB-LZD5-RCW-EN
- ZB-LZD7-RCW-MS
- ZB-LZD10-RCW-EN
- ZB-LZD15-RCW-EN

I only have the 10W version, but are you feeling safe to use `ZB-LZD-RCW` four all four variants ? Hoping all four use the same manufacturer  `_TZ3210_s9lumfhn`
Or do you expect different manufacturers for the different lamps ?

I can go for safe, and make it:
```
            tuya.whitelabel('Moes', 'ZB-LZD10-RCW-EN', '10W RGB+CCT Smart Downlight', ['_TZ3210_s9lumfhn']),
```

Please, any advice or recommendations are appreciated 

Theo

Comment (vanHoesel) [reply src/devices/tuya.ts]: I have update the file, so you can merge it if you think that is the right wat to deal with the minor problem I described above.
I can easily update the MR again

**Comment by v-ar** [comment]
How to replace wrong picture on Moes ZB-LZD10-RCW page?  I have attached the correct picture
moes-zb-lzd10-rcw

**Comment by Koenkk** [comment]
To submit a new picture
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `ZB-LZD10-RCW.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Comment by v-ar** [comment]
> To submit a new picture
> 
> 1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
> 2. Go to the `public/images/devices` directory, _Add file_ -> _Upload files_
> 3. Upload the files and press _Commit changes_
> 4. Press _Contribute_ -> _Open pull request_ -> update title/description -> _Create pull request_
> 
> Make sure that:
> 
> * The filename is `ZB-LZD10-RCW.png`
> * The size is 512x512
> * The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

Thanks, Koenkk! I'll wait for the changes 🤝

**Pull request merged**